addon/-private/system/store.js

```diff
@@ -3438,14 +3433,8 @@ function _modelForMixin(store, normalizedModelName) {
   // container._registry = 1.11 - 2.0
```

Comment:
These comments should not be needed, as we are explicitly dropping support for Ember < 2.12 here.